rules/windows/discovery_post_exploitation_external_ip_lookup.toml

```diff
@@ -54,7 +54,10 @@ network where network.protocol == "dns" and
         "*myipaddress.com",
         "*showipaddress.com",
         "*whatismyipaddress.com",
-        "*wtfismyip.com"
+        "*wtfismyip.com",
```

Comment:
This is a different rule, this file should be removed.

Reply:
Okay I revert that specific commit.